meson.build

```diff
@@ -67,19 +69,23 @@ if host_machine.system() == 'windows'
   cargs += '-DCMS_DLL_BUILD=1'
 endif
 
-liblcms2_lib = library('lcms2', lcms2_srcs,
-  include_directories : inc_dirs,
+liblcms2_lib = library(
+  'lcms2',
+  lcms2_srcs,
+  include_directories: inc_dirs,
   gnu_symbol_visibility: 'hidden',
-  dependencies: deps,
+  dependencies: [m_dep, thread_dep],
   c_args: cargs,
+  name_prefix: host_machine.system() == 'windows' ? '' : 'lib',
   version: library_version,
+  soversion: host_machine.system() == 'windows' ? '' : '2',
```

Comment:
Is it possible to make this MSVC specific somehow? Isn't this automatically figured out depending on platform like it is for CMake?

With current MinGW build using Autotools the default result is `liblcms2-2.dll`, i.e. it has both the prefix and the soversion. It would be good to keep this behaviour so all the dependant binaries don't have to be rebuilt.

Reply:
It is but the def file is specific about the name. AFAIK if it mismatches, linking won't work properly.

Reply:
The def file is completely optional, there is no need for it when you have the import library stub (at least when we talk about MSVC and MinGW).

Reply:
Sure

Reply:
FYI, if I remove those two lines, the Meson build creates identical artifacts as [Autotools for MinGW](https://packages.msys2.org/package/mingw-w64-ucrt-x86_64-lcms2?repo=ucrt64) (minus the tools and the static lib):

```
Installing liblcms2-2.dll to C:/msys64/opt/lcms2/bin
Installing liblcms2.dll.a to C:/msys64/opt/lcms2/lib
Installing C:/msys64/home/kmilos/Little-CMS/include/lcms2.h to C:/msys64/opt/lcms2/include/
Installing C:/msys64/home/kmilos/Little-CMS/include/lcms2_plugin.h to C:/msys64/opt/lcms2/include/
Installing C:/msys64/home/kmilos/Little-CMS/build/meson-private/lcms2.pc to C:/msys64/opt/lcms2/lib/pkgconfig
```

No .def file in either case.